plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1185,7 +1196,6 @@ enum HostNicType {
         private String _sourceName;
         private String _networkName;
         private String _macAddr;
-        private String _ipAddr;
```

Comment:
Why is this being removed?

Reply:
I think it was not used / set anywhere. I will add it back however.

Reply:
why add it back if it is not used?